src/emulator/modules/SceTouch/SceTouch.cpp

```diff
@@ -55,16 +137,22 @@ static int peek_touch(HostState &host, SceUInt32 port, SceTouchData *pData, SceU
 
         if ((touch_pos_viewport.x >= 0) && (touch_pos_viewport.y >= 0) && (touch_pos_viewport.x < 1) && (touch_pos_viewport.y < 1)) {
             pData->report[pData->reportNum].x = static_cast<uint16_t>(touch_pos_viewport.x * 1920);
-            pData->report[pData->reportNum].y = static_cast<uint16_t>(touch_pos_viewport.y * 1088);
+            if (port == SCE_TOUCH_PORT_FRONT) {
+                pData->report[pData->reportNum].y = static_cast<uint16_t>(touch_pos_viewport.y * 1088);
+            } else {
+                pData->report[pData->reportNum].y = static_cast<uint16_t>(108 + touch_pos_viewport.y * (889 - 108));
```

Comment:
Why 889 - 108? Also what's that 108 value?

Reply:
i left it as 889 - 108 too be easier to read, but i realise now thats an extra calculation each cycle. 108 is minAaY and 889 is maxAaY for the back panel. If you just use the front display sizes the back touchscreen is offset as it gets closer to the edges.

Reply:
No extra cycle, the compiler will be smart enough to calculate it at compiletime

Reply:
good to know, i did change it though before i read this

Reply:
parhentesis should be removed if you're going to remove the 889 - 108. Btw it was just curiosity in why we had this 889 and 108 (they looked like some magic random numbers to me), not really a change request.

Reply:
aghhh...forgot the parenthesis ill change them now and push

Reply:
done